Workbooks/Workloads/Redis Insights/Redis Insights.workbook

```diff
@@ -669,10 +681,10 @@
               "size": 0,
               "aggregation": 3,
               "showAnalytics": true,
-              "title": "Connected Slaves",
+              "title": "Connected slaves",

```

Comment:
We should probably rename this to Connected replicas.

Reply:
done